Makefile

```diff
@@ -40,3 +39,19 @@ rpm:
 		printf "\nThe tendrl-api RPMs are located at:\n\n"; \
 		printf "   $(DEPLOY)/latest\n\n\n\n"; \
 	fi
+
+srpm:
+	@echo "target: rpm"
+	@echo  "  ...building rpm $(V_ARCH)..."
+	rm -fr $(BUILDS)
+	mkdir -p $(DEPLOY)/latest
+	mkdir -p $(RPMBUILD)/SPECS
+	sed -e "s/@VERSION@/$(VERSION)/" tendrl-api.spec \
+	        > $(RPMBUILD)/SPECS/tendrl-api.spec
```

Comment:
Here you replace string `@VERSION@` with value from `VERSION` variable, but there is no such string anywhere but in the `Makefile`:

```
$ git grep '@VERSION@'                                                         
Makefile:       sed -e "s/@VERSION@/$(VERSION)/" tendrl-api.spec \             
Makefile:       sed -e "s/@VERSION@/$(VERSION)/" tendrl-api.spec \  
```